assignments/homework/Raymond Ingram/data/2mass.tbl

```diff
@@ -0,0 +1,1000 @@
+\fixlen = T
```

Comment:
reuse the data files from the class directory, no need to make another copy of them in the repo. You can access it using  relative path from your python file